port/common/omrthread.c

```diff
@@ -0,0 +1,41 @@
+#if defined (LINUX)
+#define _GNU_SOURCE
+#include <sys/resource.h>
+#endif /* defined (LINUX) */
+
+#include <stdlib.h>
+
+#include "omrport.h"
+#include "omrportpriv.h"
+#include "omrthread.h"
+
+int32_t omrthread_get_self_thread_time(struct OMRPortLibrary *portLibrary, omrthread_thread_time_t *threadTime) {
+	omrthread_t self = omrthread_self();
+
+	int64_t cpuTime = omrthread_get_cpu_time(self);
```

Comment:
in the linux case I think its better to get system and user time from `rUsage`

Reply:
Done.